bats/fb-content-katello.bats

```diff
@@ -10,91 +10,97 @@ setup() {
   tSetOSVersion
 }
 
+@test "Create an Empty Organization" {
+  run hammer -u admin -p changeme organization info --name "Empty Organization"
```

Comment:
Yea - otherwise this was failing the test when really I just want the output and status to use for the real test.